### ISSUE TITLE
Failing to install transitive dep with 0.15-beta ("Registry _@npm not found")

### ISSUE THREAD

**Issue by theefer** [jspm/jspm-cli#618]
The following command fails with jspm 0.15 beta in a fresh directory:

```
$ jspm install npm:pandular

     Looking up npm:pandular
     Updating registry cache...

err  Registry panda-session@npm not found.

err  Unable to load registry panda-session@npm

warn Installation changes not saved.
```

whereas it worked fine with jspm 0.14.0.

**Comment by guybedford** [comment]
Thanks for testing, will post a patch soon.

**Comment by guybedford** [comment]
Fixed in 0.15.0.
